lib/pf/db.pm

```diff
@@ -177,7 +177,7 @@ sub get_db_handle {
 
 sub db_data {
     my ($from_module, $module_statements_ref, $query, @params) = @_;
-    my $timer = pf::StatsD::Timer->new({ 'stat' => called() . ".$query", sample_rate => 0.1});
+    my $timer = pf::StatsD::Timer->new({ 'stat' => called() . ".$query", sample_rate => 0.1, level => 9});
```

Comment:
Maybe a bit high here? 
We may want some of that good stuff without going all the way to 11.